src/vstest.console/Processors/EnableCodeCoverageArgumentProcessor.cs

```diff
@@ -155,11 +155,12 @@ internal class EnableCodeCoverageArgumentExecutor : IArgumentExecutor
 @"            </Functions>" + Environment.NewLine +
 @"            <Attributes>" + Environment.NewLine +
 @"              <Exclude>" + Environment.NewLine +
-@"                <Attribute>^System.Diagnostics.DebuggerHidden.*</Attribute>" + Environment.NewLine +
-@"                <Attribute>^System.Diagnostics.DebuggerNonUserCode.*</Attribute>" + Environment.NewLine +
-@"                <Attribute>^System.Runtime.CompilerServices.CompilerGenerated.*</Attribute>" + Environment.NewLine +
-@"                <Attribute>^System.CodeDom.Compiler.GeneratedCode.*</Attribute>" + Environment.NewLine +
-@"                <Attribute>^System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverage.*</Attribute>" + Environment.NewLine +
+@"                <Attribute>^System.Diagnostics.DebuggerHiddenAttribute$</Attribute>" + Environment.NewLine +
+@"                <Attribute>^System.Diagnostics.DebuggerNonUserCodeAttribute$</Attribute>" + Environment.NewLine +
```

Comment:
What is issue with System.Diagnostics.DebuggerNonUserCode.*? Is it looking for System.Diagnostics.DebuggerNonUserCodeXYZAttribute? What was the error?